### ISSUE TITLE
Score: Active vs Quiet Wake.

### ISSUE THREAD

**Issue by dlevenstein** [buzsakilab/buzcode#299]
This has been on the potential to do list for a while and I'm thinking about implementing it.... First pass would be to use the theta threshold from REM to divide active/quiet wake. Anyone with thoughts?

**Comment by dlevenstein** [comment]
Bottom right plot shows the occupancy heat map for the non-NREM time..... notice that the wake cloud is continuous, but could be potentially divided by the same threshold that separates out REM.

I could add this in as an option to try out and see how it goes?

@brendonw1 ?

![screenshot - 02282019 - 03 18 22 pm](https://user-images.githubusercontent.com/7265688/53595953-9b542300-3b6c-11e9-80e4-79bde39abe7e.png)

**Comment by AntonioFR8** [comment]
Theta/delta power ratio plus movement from accelerometer or tracking will be good enough to separate both states

**Comment by dlevenstein** [comment]
The theta used by SleepScoreMaster is (5-10Hz)/(2-20Hz). 
("narrowband theta" above, we found that to be more reliable than theta/delta, because it's isolated narrowband theta on both sides, Gervasoni did something similar)

SSM doesn't use accelerometer because it's not available in all recordings, but the EMG signal is pretty good (see above, bottom right panel). It's not cleanly bimodal, but it looks like the threshold from REM could be used.

**Comment by dlevenstein** [comment]
The only frustrating thing is the small v high EMG mode, which tends to be associated with microarousals, but is not present in all recordings

**Comment by AntonioFR8** [comment]
Yes, it looks good. Perhaps is worthy to add an option to read out the accelerometer signals and most of the recordings we do now have it. 
If you implement it in the SSM I will test it in a session in which I already classify active / quiete wake

**Comment by brendonw1** [comment]
Sure!
Wanna call it state 2 ("drowsy")?... between 1:Wake and 3:NREM

Thanks

**Comment by dlevenstein** [comment]
Definitely state 2, though I think in the StateScore output I'd make it QWake or QuietWake or something like that? Not sure I'd like to commit to the animal being drowsy.

Plan is to make it a checkable box in TheStateEditor and input option in SleepScoreMaster (default: false, for now)

**Comment by dlevenstein** [comment]
Re: accelerometer - at this point, the SSM is well optimized for the LFP-derived EMG signal and it works well. I'd rather not go through the process of optimizing it for another optional signal which will have different statistics (for example, may not be so cleanly bimodal (middle left panel above) but may have a heavy tail). We think the EMG likely reflects muscle tone in stabilization muscles as well as movement, so it's very good for REM/NREM/MA segregation in a way that the accelerometer may or may not be.

**Comment by brendonw1** [comment]
Sorry I didn't mean to call it drowsy... I agree I wouldn't... was just specifying what it has been called as a name.  I think QuietWake would be good

**Comment by dlevenstein** [comment]
👍  Number 2 it is!

Maybe one of these days we’ll get our intermediate sleep detection and fill our all 5 numbers in TheStateEditor

**Comment by brendonw1** [comment]
Did I ever tell you - actually Karim Benchenane and i talked about IS and I
think we score it differently.  Basically he calls it REM and we call it
part of nonREM.  Interesting. In his sleep staging paper/biorxiv he gets
some different results than us and he was thinking it resulted from that
difference about when you say REM starts

**Comment by dlevenstein** [comment]
Manu has gotten this to work - we'll work together to integrate into SleepScoreMaster
